.codecov.yml

```diff
@@ -6,9 +6,10 @@ coverage:
   status:
     project:
       default:
-        threshhold: 5%
+        target: 90%
```

Comment:
This change appears to be unrelated to the PR, as it is not mentioned in the description.
Is this a merge problem?

Reply:
It is the same change as is in #1023 to correctly control the limits.

I will add something to the description to mention it.